Packs/FireEyeETP/Integrations/FireEyeETP/FireEyeETP.yml

```diff
@@ -300,6 +300,62 @@ script:
     - contextPath: FireEyeETP.Alerts.id
       description: The alert unique ID
     description: Detailed information from any particular alert. Alerts more than 90 days old are not available.
+  - name: fireeye-etp-download-alert-artifact
+    arguments:
+    - name: alert_id
+      required: true
+      description: The alert id
+    description: 'Downloads all case files of the alert specified by alert ID, in
+      a zip file. You can obtain the ID from the Alert Summary response, for example
+      "id": "AV7zzRy7kvIwrKcfu0I".'
+  - name: fireeye-etp-list-yara-rulesets
+    arguments:
+    - name: policy_uuid
+      required: true
+      description: Universally unique identifier (UUID) of the policy. (Can be found
+        on the URL of the ETP Policies)
+    outputs:
+    - contextPath: FireEyeETP.Policy
+      description: The policy id.
+    description: Fetch the list of YARA rulesets and return a list with all the rules.
```

Comment:
Can you please add more relevant outputs if you think they are important?
you can see examples in: fireeye-etp-search-messages command

Reply:
we do not see any other output as relevant

Reply:
ok great thank you!